yarn.lock

```diff
@@ -2072,6 +2072,21 @@ atob@^2.1.2:
   resolved "https://registry.yarnpkg.com/atob/-/atob-2.1.2.tgz#6d9517eb9e030d2436666651e86bd9f6f13533c9"
```

Comment:
This is an open question for all the team, are we going to share this .lock file, or might be better to ignore it?

Reply:
what do you suggest we do with it? @Alxzu @martinjaimem

Reply:
Keep it, https://github.com/yarnpkg/yarn/issues/1583#issuecomment-257544217